commonItems/UnitTests/CommonFunctionsTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace commonItems.UnitTests {
+    public class CommonFunctionsTests {
```

Comment:
Reorder these to match the order of the function they're testing.

Reply:
This is no longer relevant with the splitting up of CommonFunctionsTests.